Doc/howto/descriptor.rst

```diff
@@ -1533,31 +1567,44 @@ At this point, the metaclass has loaded member objects for *x* and *y*::
 
     # We test this separately because the preceding section is not
     # doctestable due to the hex memory address for the __init__ function
-    >>> isinstance(vars(H)['x'], Member)
+    >>> isinstance(H.x, Member)
```

Comment:
No thanks.  I'm specifically relating this code back to the listing of what is in the instance dict.

Reply:
Alright.